langwatch/src/server/tracer/opentelemetry.test.ts

```diff
@@ -565,11 +565,7 @@ const fastApiOpenTelemetryRequest: DeepPartial<IExportTraceServiceRequest> = {
                 {
                   key: "http.status_code",
                   value: {
-                    intValue: {
-                      low: 404,
-                      high: 0,
-                      unsigned: false,
-                    } as any,
```

Comment:
this example was here because I did get such case, I'd rather we keep at least one of this format

Reply:
oh weird, i removed as otel spec doesn't support it. imo we should reject non-spec behaviour, but if it was an important integration I can add back?